package sinhee.kang.tutorial.global.exception.exceptions.conflict


class UserAlreadyExistsException: ConflictException("User is Already Exists")
